### PR TITLE
support for server blocks

### PR DESCRIPTION
**Pull request by tballmsft**

### CODE REVIEW

File: src/servers/rolemanager.ts [hunk 1]
```diff
@@ -14,6 +14,7 @@ import { serviceSpecificationFromName } from "../jdom/spec"
 export interface RoleBinding {
     role: string
     serviceShortId: string
+    client: boolean
```
Comment (pelikhan) [inline src/servers/rolemanager.ts:17]: The role manager as it is is still compatible with the service specification, so we can use it as a simulator. What's the goal of this change?

Comment (tballmsft) [reply src/servers/rolemanager.ts:17]: Roles are, by default, for programming against (client = true), no change in behavior. When we are doing server blocks, we need roles that we are implement.

Comment (tballmsft) [reply src/servers/rolemanager.ts:17]: You are supposed to be on vacation...

Comment (tballmsft) [reply src/servers/rolemanager.ts:17]: actually I figured out a way that doesn't need this change to role blocks

**Pull request closed without merging**